modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -12,7 +12,7 @@ def config
         openapi: DocHelpers.wip_doc_enabled?(:segmented_apis) ? '3.1.0' : '3.0.0',
```

Comment:
I left almost all of the rswag config and files (for example, the markdown descriptions and the generated json) in their old (v2) folders. I think it would make more sense to change these to v0 as well (so that only decision reviews v2 stuff is left in folders with 'v2' in the name), but I felt like it warranted a bit of discussion beforehand, since it seems like it would really significantly change the rswag configs - I put it on the agenda for our engineering syncs.